site/validate.md

```diff
@@ -60,18 +60,7 @@ The output of the tool will look something like this:
       Errors: 1
         Message: Machines is not one of the folder, folder instance or attribute names allowed in model location topology:/Server/AdminServer
 
-To get the valid list of attributes and folders at this model location, run the tool as follows:
-
-    weblogic-deploy\bin\validateModel.cmd -oracle_home c:\wls12213 -print_usage topology:/Server
-
-This will print out the list of attributes and valid subfolders (the full output is omitted here for brevity) that will include the following attribute in the list:
-
-    Section: topology:/Server
-
-      Valid Attributes are :-
-        ...
-        Machine                                             string
-        ...
+The [Model Help Tool](model_help.md) can be used to determine the valid the valid list of attributes and folders at this model location.
```

Comment:
The [Model Help Tool](model_help.md) can be used to determine -> Use the [Model Help Tool](model_help.md) to determine
the valid the valid list -> the valid list